package.json

```diff
@@ -55,11 +57,13 @@
     "eslint-plugin-promise": "^4.2.1",
     "eslint-plugin-standard": "^4.0.1",
     "mocha": "5.2.0",
+    "mongodb": "^3.5.4",
```

Comment:
I think given the number of `ObjectId()` calls we have we might run the risk of running out of entropy, but yes something like this is probably better.

Reply:
True.  How about `crypto.createHash('md5').update(`${counter++}`).digest("hex").slice(0,24)`?

Reply:
Pinched this from SO which seems to work just fine
https://github.com/overleaf/filestore/pull/86/commits/f3d503cb0fbc62cebad198385ce5f2c010026d62

Reply:
Thanks, I have learned some things about object IDs due to this discussion :)

While I am generally in favour of not having too many dependencies... I feel like having to jump through these hoops and/or worry about running out of entropy is probably enough of a benefit that we should keep the dependency. It's server-side only, and it's something we depend on all over, so I don't think it is particularly bad.

If we do the planned (or at least being considered?) work to ship only production dependencies instead of both dev and production dependencies, it would also affect only dev.

Reply:
Happy to drop the commit